ui/page/send/page.go

```diff
@@ -218,7 +218,21 @@ func (pg *Page) initializeAccountSelectors() {
 				// only mixed accounts can send to address/wallets for wallet with privacy setup
 				// don't need to check account the same with destination account
 				accountIsValid = account.Number == load.MixedAccountNumber(pg.selectedWallet)
+
+				// For an Intra-Accounts to happen transfer the bare minimum expected is that:
+				// 1. There is only one recipient instance available.
+				// 2. Both (i.e. source and recipient) use the same wallet.
+				// 3. Source account selected must have a spendable balance
+				// 4. Recipient's wallet tab option must be active.
```

Comment:
this no.4 isn't clear what it means

Reply:
There are two tabs on the recipient info; "Address" and "Wallets". To activate the "wallets" tab, it needs to be currently on display.